### PR TITLE
Add original content to API

### PR DESCRIPTION
**Pull request by ketkarameya**
The PR basically makes available - the content of the file before the rewrite. 

--------
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #413

### CODE REVIEW

File: src/tests/test_piranha_java.rs [hunk 1]
```diff
@@ -122,17 +122,18 @@ fn test_new_line_character_used_in_string_literal_code_snippet() {
...
     path_to_configurations = path_to_scenario.join("configurations").to_str().unwrap().to_string(),
     language = PiranhaLanguage::from(JAVA),
     dry_run = true,
-    code_snippet = "package com.uber.piranha;
-    class SomeClass {
-      void someMethod(String s) {
-        assert (s.equals(\"Hello \\n World\"));
-      }
-    }".to_string(),
+    code_snippet = code_snippet.to_string(),
```
Comment (rajbarik) [inline src/tests/test_piranha_java.rs:136]: You are doing to_string twice. See line 131.

File: src/models/piranha_output.rs [hunk 2]
```diff
@@ -45,25 +53,10 @@ impl PiranhaOutputSummary {
...
     return PiranhaOutputSummary {
       path: String::from(source_code_unit.path().as_os_str().to_str().unwrap()),
+      original_content: source_code_unit.original_content().to_string(),
       content: source_code_unit.code().to_string(),
       matches: source_code_unit.matches().iter().cloned().collect_vec(),
       rewrites: source_code_unit.rewrites().iter().cloned().collect_vec(),
     };
   }
-
-  pub(crate) fn matches(&self) -> &[(String, Match)] {
```
Comment (rajbarik) [inline src/models/piranha_output.rs:54]: What happened to these functionalities?

Comment (ketkarameya) [reply src/models/piranha_output.rs:54]: We dont need them. These have been replaced by the annotation `#[get = "pub(crate)"]` which is added to corresponding property.

**Review by ketkarameya — COMMENTED** [review]
Addressed review comments https://github.com/uber/piranha/pull/413/commits/503fc6fcf2f710bd360f22e6189ad6ca9e6b3d1e

**Review by rajbarik — APPROVED** [review]

**Pull request merged**